useful-scripts/misc-test-utils/src/modules/publication/publish/index.ts

```diff
@@ -7,21 +7,24 @@ import StreamZip from 'node-stream-zip';
 import globby from 'globby';
 import rimraf from 'rimraf';
 import FormData from 'form-data';
-import { ReleaseDataProps, SubjectArrayT } from 'types';
-import ZipDirectory from './utils/ZipDirectory';
-import Sleep from './utils/Sleep';
+import ZipDirectory from '../../../utils/zipDirectory';
+import Sleep from '../../../utils/Sleep';
+import { SubjectArray } from '../../../interfaces/SubjectArray';
+import { ReleaseData } from '../../../interfaces/ReleaseData';
+import errorHandler from '../../../utils/errorHandler';
+import chalk from 'chalk';
 
 // disable insecure warnings
 process.env.NODE_TLS_REJECT_UNAUTHORIZED = '0';
-
-const { JWT_TOKEN, API_URL, TOPIC_ID } = process.env;
+const { ADMIN_URL, TOPIC_ID, JWT_TOKEN } = process.env;
+const cwd = process.cwd();
```

Comment:
I guess this is intended but is it a good idea to dump stuff into the current directory?

The main problem is that can potentially dump the files outside of the sub-project directory if you run the script outside of it. This is probably something we want to avoid? 

I'm guessing ideally we want the test results to go into `importer-publisher-tests/test-results`? If so, I would suggest setting up some constants file (perhaps in the sub-project root), which sets up variables for things like the intended test result output directory. We can then import this into the required files for use.

Reply:
I don't think the test results directory is being used. I'll remove it as part of this PR. The whole cli will break if you attempt to start it outside the root of the `misc-test-utils` directory since it looks for a zipped data file in whatever the current working directory is (which we assume is `misc-test-utils`).